packages/react-router/lib/dom/ssr/routes.tsx

```diff
@@ -39,7 +39,7 @@ export interface EntryRoute extends Route {
   hasErrorBoundary: boolean;
   imports?: string[];
   css?: string[];
-  module: string;
+  module?: string; // Undefined for <HydratedRouter routes>
```

Comment:
This needs to be optional for routes coming from `<HydratedRouter routes>` so had to make some other minor changes to existing code to account for that